### ISSUE TITLE
Is contrast high enough?

### ISSUE THREAD

**Issue by tripu** [w3c/wbs-design#53]
I have received these comments about the new colours:

> *&ldquo;Black over violet with lowercase lacks enough contrast with that font. Try to adjust either the font or the colours. This is also the case for the questions, like* &lsquo;1. Decision&hellip;&rsquo; *black over a semi-black blue-indigo. That is not the best contrast.&rdquo;*

Is that person right?

**Comment by lcarcone** [comment]
Maybe the contrast for the question titles is too low, let's check it.
Concerning the violet color, it is defined in the w3c template, we can't change it. I don't see lowercase characters over violet color, do you know which page the person is referring to ?

**Comment by tripu** [comment]
I'm guessing the &ldquo;blue&rdquo; is `#b3cdf3`; and the &ldquo;violet&rdquo;, either `#f5f5ff` or `#f8efff`&hellip; But I'm not sure (I'll try to find out).

**Comment by lcarcone** [comment]
Yes, 'blue' is #b3cdf3 and color #111 so the contrast ratio should be 11,64, higher than the minimum defined in https://www.w3.org/TR/WCAG21/#contrast-minimum
Same for 'violet' #f8efff and color #000, ratio is 18,78
I remember a remark from Michael about contrast with the icons, I'll check.  And maybe there are other parts of the questionnaire where the contrast is not enough.

**Comment by wseltzer** [comment]
Relatedly, the pastel colors are UGLY, especially the purple.

**Comment by lcarcone** [comment]
Hi Wendy,

I don't like it very much either.
We did not choose it, it has been chosen for the member pages in 2008, I guess it has been validated at the time.
For wbs, I would like to switch the public blue color to the member pages and use another color for the public pages, there are very few public pages compared to member pages. There is a risk that the users will be confused, but I think it's worth trying.
About the overall design, the goal was not to have a specific design for wbs but rather to be consistent with the w3c site. Marcomm is working on a new template for the w3c site, when it is ready we'll apply it to wbs.

Thanks,
Laurenrt
